include/sound/soc-dpcm.h

```diff
@@ -143,11 +161,11 @@ static inline void soc_dpcm_debugfs_add(struct snd_soc_pcm_runtime *rtd)
 }
 #endif
 
-int dpcm_path_get(struct snd_soc_pcm_runtime *fe,
-	int stream, struct snd_soc_dapm_widget_list **list_);
+int dpcm_path_get(struct snd_soc_pcm_runtime *fe, int stream,
+	struct snd_soc_dapm_widget_list **list_);
 void dpcm_path_put(struct snd_soc_dapm_widget_list **list);
-int dpcm_process_paths(struct snd_soc_pcm_runtime *fe,
-	int stream, struct snd_soc_dapm_widget_list **list, int new);
+int dpcm_process_paths(struct snd_soc_pcm_runtime *fe, int stream,
+	struct snd_soc_dapm_widget_list **list, bool new, bool force_prune);
```

Comment:
This was reviewed on alsa-devel, so I'll just add this as a remark but multiple booleans in function arguments is pushing limits of readability. The newly added enum reads better to convey the requested action.

Reply:
@kv2019i mmh, how are multiple booleans worse than multiple integers or anything else? :-) If you look inside `dpcm_process_paths()` you'll see that those two booleans are handled absolutely independently, they really aren't related, don't think an enum would make sense here.

Reply:
@lyakh don't have problems with booleans, but for some reason the name "new" bothers me :), so "new" what? also plain "new" brings me recollections from c++ etc...

Reply:
@juimonen oops, you're right, I keep forgetting about c++ :-) BTW, this PR is way outdated. We're back to #1501. But don't worry, the same comment applies there, so, your work wasn't in vain! But please continue there :-)